ui/vite.config.js

```diff
@@ -4,10 +4,44 @@ import { defineConfig } from "vite";
 import vue from "@vitejs/plugin-vue";
 import monacoEditorPlugin from "vite-plugin-monaco-editor";
 
+const streamsyncComponentsDocs = () => {
+	let config;
+
+	return {
+		name: "vue-docs-ignore",
```

Comment:
Is `vue-docs-ignore` descriptive? Aren't we transforming rather than ignoring? Please correct me if I'm wrong.